std/path.d

```diff
@@ -2504,33 +2506,103 @@ unittest
 string relativePath(CaseSensitive cs = CaseSensitive.osDefault)
     (string path, lazy string base = getcwd())
```

Comment:
I don't think it'll improve things, besides, relativePath() is now a legacy function. Those who want to control memory use should use toRelativePath().